src/observability/agg-summary.json

```diff
@@ -0,0 +1,223 @@
+{
+    "Positive test 1": [
```

Comment:
It is best to put these test input/output files under `test` directory in the root, instead of mixing this with source code.